src/main/java/io/github/moulberry/notenoughupdates/profileviewer/level/task/SkillRelatedTaskLevel.java

```diff
@@ -217,6 +217,10 @@ private static double powder(double multiplier, float left, int CAP) {
 		double cons = 1758267;
 		if (left <= 0) return 0;
 
+		if(left >= CAP) {
+			left = CAP-350_000;
```

Comment:
Wouldn't this currently give people just below `CAP` more exp than those exactly at `CAP` (because the if check also doesnt subtract `350000`

Reply:
left is already subtracted with 350k

Reply:
that doesnt change anything: 

```java
int left1 = 1349000;
int left2 = 1350001;
int cap = 1000000;
left1 -= 350000;
left2 -= 350000;

if ( left1 >= CAP ) {
	left1 = CAP - 350000;
}
if (left2 >= CAP) {
	left2 = CAP - 350000;
}
// left1 = 999000;
// left2 = 650000;
```

Reply:
i think i get you ill think about it again tomorrow